search/search-client-query-transformer/src/main/java/io/camunda/search/clients/SearchClients.java

```diff
@@ -69,11 +70,19 @@ public class SearchClients
 
   public SearchClients(
       final DocumentBasedSearchClient searchClient,
-      final boolean isCamundaExporterEnabled,
-      final String prefix) {
+      final IndexDescriptors indexDescriptors,
+      final boolean isCamundaExporterEnabled) {
     this(
         searchClient,
-        ServiceTransformers.newInstance(isCamundaExporterEnabled, prefix),
+        ServiceTransformers.newInstance(
+            indexDescriptors,
+            indexDescriptor -> {
+              final String alias = indexDescriptor.getAlias();
+              if (!isCamundaExporterEnabled && "tasklist-task-8.5.0_alias".equals(alias)) {
+                return "tasklist-list-view-8.6.0_alias";
+              }
+              return alias;
+            }),
```

Comment:
This will eventually [go away](https://camunda.slack.com/archives/C05QBRGMA9F/p1732539829888809).